lib/src/pubspec.dart

```diff
@@ -293,8 +298,18 @@ class Pubspec extends PubspecBase {
           pubspecPath);
     }
 
+    String? overridesContents;
+    final overridesPath = path.join(packageDir, 'pubspec_overrides.yaml');
```

Comment:
Can we not make `Pubspec.load` take `pubspecOverridesFile` as an optional argument?

Reply:
Done